portfolio/src/main/webapp/script.js

```diff
@@ -15,7 +15,7 @@
 /** Chooses a random cat photo to display on page. */
 function changeCatPhoto() {
   const photos =
-      ["/images/cats-1.jpg", "/images/cats-2.jpg", "/images/cats-3.jpg", "/images/cats-4.JPG", "/images/cats-5.jpg"];
+      ["/images/cats-1.jpg", "/images/cats-2.jpg", "/images/cats-3.jpg", "/images/cats-4.JPG", "/images/cats-5.jpg", "/images/cats-6.jpg", "/images/cats-7.jpg"];
```

Comment:
Javascript (and most languages) have a column width of max 80 characters. So we should break this into separate lines, ex:

["/images.cats-1.jpg",
 "/images.cats-2.jpg",
...]

https://google.github.io/styleguide/jsguide.html#formatting-column-limit

Depending on the editor/IDE you're using to code in, it might automatically do this for you or at least warn you with Linters/fix commands. But sometimes it's hard to notice (Github UI also makes it hard to see, so I just eyeballed it and noticed it might be more than 80 chars haha)